.travis.yml

```diff
@@ -52,7 +52,11 @@ addons:
 
 # For MacOSX systems
 before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then HOMEBREW_NO_AUTO_UPDATE=1 brew install xz || ( brew update && brew install xz ); fi
+  - |
+    if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+      HOMEBREW_NO_AUTO_UPDATE=1 brew install xz || \
+      ( brew update && brew install xz )
+    fi
```

Comment:
Rather than reformatting this bit, you could just nuke the whole `before_install` stanza as brewed xz is no longer necessary on macOS, configure or no configure.

Reply:
I'll add an additional commit to nuke it, so we can see if travis still works.  I assume they're reasonably up to date.